contracts/deposit-service/AxelarDepositService.sol

```diff
@@ -240,6 +248,25 @@ contract AxelarDepositService is Upgradable, DepositServiceBase, IAxelarDepositS
         refundToken = address(0);
     }
 
+    function refundLockedAsset(
```

Comment:
All good but it feels a little scary that the refund issuer has so much power that it can send however amount to wherever it wants to.

Reply:
since it's unlikely that we would need to refund from the deposit service often, maybe we should only let the owner collect the funds? @re1ro

Reply:
ok, yeah, initially a bunch of apps won't set the refund address (since it'll take some time to work it in the UI), so I can see it being useful to have it be a hot wallet instead. and we can swap that out later to a cold wallet if needed.